Tests/LifecycleTests/ServiceLifecycleTests.swift

```diff
@@ -226,4 +227,10 @@ final class ServiceLifecycleTests: XCTestCase {
         XCTAssertEqual("\(ServiceLifecycle.Signal.INT)", "Signal(INT, rawValue: \(ServiceLifecycle.Signal.INT.rawValue))")
         XCTAssertEqual("\(ServiceLifecycle.Signal.ALRM)", "Signal(ALRM, rawValue: \(ServiceLifecycle.Signal.ALRM.rawValue))")
     }
+
+    func testBacktracesInstalledOnce() {
+        let config = ServiceLifecycle.Configuration(installBacktrace: true)
+        _ = ServiceLifecycle(configuration: config)
+        _ = ServiceLifecycle(configuration: config)
```

Comment:
duplicate?

Reply:
deliberate... making sure second instantiation does not trigger another setup of backtraces

Reply:
i see. maybe add a code comment.